### DISCUSSION TITLE
有推进主线的功能吗?

### DISCUSSION THREAD

**Discussion by A5yncX** [MaaAssistantArknights/MaaAssistantArknights#8783] [Q&A]
目前只用来挂肉鸽, 有推进主线的功能吗?

**Comment by Constrat** [comment]
For that you go to https://prts.plus/ select the jobs for the needed stages and turn on these two options. Add your jobs one by one and Start.
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56174894/88e3621a-20fb-439f-b913-45e3497fe974)
